include/zephyr/lorawan/lorawan.h

```diff
@@ -330,6 +332,24 @@ void lorawan_get_payload_sizes(uint8_t *max_next_payload_size,
  */
 int lorawan_set_region(enum lorawan_region region);
 
+/**
+ * @brief Request for time according to DeviceTimeReq MAC cmd
+ *
+ * Append MAC DevTimeReq command. It will be processed on next send
+ * message or force sending empty message to request time immediately.
+ *
+ * @param force_request Force sending an empty message execute request
```

Comment:
The following seems clearer to me:
`@param force_request Immediately send an empty message to execute the request`